packages/nx/src/native/tests/command.spec.ts

```diff
@@ -1,3 +1,4 @@
+import { checkUnknownRules } from 'ajv/dist/compile/util';
```

Comment:
I think this was added by accident. Lets remove it

Reply:
Yeah, thanks